src/Containers/Settings/AccountCard.tsx

```diff
@@ -38,7 +41,7 @@ export const AccountCard: React.FC<AccountProps> = ({ employee, logout }) => {
             />
             <Button
                 margin="20px 0 0"
-                onClick={() => logout()}
+                onClick={(): void => logout()}
```

Comment:
Just pass logout, since it's just a function reference no need to do 
() => func()
You can do
{logout}
When you are not passing any parameters btw

Reply:
@ralph-dev 

I got a TS error passing the function as a reference so I changed the AccountProps interface from `logout: Function` to `logout: (event: React.MouseEvent<HTMLButtonElement>) => void` 

https://github.com/cheapreats/react-ui-library/blob/v2/src/Containers/Settings/AccountCard.tsx